src/camera.js

```diff
@@ -881,103 +865,49 @@ camera.projection = {
 };
 
 /**
- * Camera clipping bounds, probably shouldn't be modified.
+ * Default camera clipping bounds.  Some features and renderers may rely on the
+ * far clip value being more positive than the near clip value.
  */
-camera.bounds = {
-  left: -1,
-  right: 1,
-  top: 1,
-  bottom: -1,
-  far: -2,
-  near: -1
+camera.clipbounds = {
+  perspective: {
```

Comment:
typically for perspective, you use angle, aspect and near far only (e.g. see here: http://www.ntu.edu.sg/home/ehchua/programming/opengl/CG_examples.html). I think setting in terms of gcs bounds is also useful so perhaps we are fine for now.

Reply:
I fully expect that the perspective mode will change further when we start using it in earnest.